src/models/device.ts

```diff
@@ -149,11 +149,10 @@ const getDeviceModel = function (
 		}),
 	);
 
-	const getOsUpdateHelper = once(async () => {
+	const getOsUpdateHelper = once(async (deviceActionsApiVersion: string) => {
```

Comment:
We can be a bit more precise with:
```suggestion
	const getOsUpdateHelper = once(async (deviceActionsApiVersion: 'v1' | 'v2') => {
```